packages/bits/spec/components/switch/switch.a11y.ts

```diff
@@ -0,0 +1,17 @@
+import { browser } from "protractor";
+import { assertA11y, Helpers } from "../../helpers";
+import { SwitchAtom } from "../public_api";
+
+describe("a11y: switch", () => {
+    let rulesToDisable: string[] = [
+        "color-contrast", // NUI-6014
+    ];
+
+    beforeAll(async () => {
+        await Helpers.prepareBrowser("switch/switch-visual-test");
+    });
+
+    it("should check a11y of switch", async () => {
+        await assertA11y(browser, SwitchAtom.CSS_CLASS, rulesToDisable);
+    });
```

Comment:
Maybe it would be good to add one more test with switch - on

Reply:
the testing page itself has the switch component in "on" state